classes/i18n/AppLocale.inc.php

```diff
@@ -1,49 +0,0 @@
-<?php
-
-/**
- * @file classes/i18n/AppLocale.inc.php
- *
- * Copyright (c) 2014-2021 Simon Fraser University
- * Copyright (c) 2003-2021 John Willinsky
- * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
- *
- * @class AppLocale
- * @ingroup i18n
- *
- * @brief Provides methods for loading locale data and translating strings identified by unique keys
- *
- */
-
-namespace APP\i18n;
-
-use PKP\i18n\PKPLocale;
-
-class AppLocale extends PKPLocale
```

Comment:
I've removed the `AppLocale` since its only purpose was to add extra locale files.
If the application needs to add something else to the `Locale`, it still can be done through a container replacement.